src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -59,6 +62,10 @@ export interface StoragePoolFormValues {
   cephfs_osd_pg_num?: string;
   cephfs_path?: string;
   cephfs_user_name?: string;
+  cephobject_radosgw_endpoint?: string;
+  cephobject_cluster_name?: string;
+  cephobject_user_name?: string;
+  cephobject_bucket_name_prefix?: string;
```

Comment:
We are missing to initialize the values on edit. The new fields here need to be added to the `toStoragePoolFormValues` function that is called by the `EditStoragePool` component.